commit 34238c257d3cb6fe7eba247c7e40a1dd49c4f3e8 (refs/changes/18/2418/2)
Author: Richard Bradley <richard.bradley@joyent.com>
Date:   2017-08-17T17:53:49+00:00 (2 years, 2 months ago)
    
    MANATEE-352 Add support for "synchronous_commit=off" in pg_overrides
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/lib/postgresMgr.js b/lib/postgresMgr.js
index 1985afd..be7f693 100644
--- a/lib/postgresMgr.js
+++ b/lib/postgresMgr.js
@@ -123,7 +123,8 @@ var TUNABLES_SCHEMA = {
                 enum: [
                     'on',
                     'remote_apply',
-                    'remote_write'
+                    'remote_write',
+                    'off'
                 ]
             }
         },
